src/promptflow-evals/promptflow/evals/evaluate/_batch_run_client/code_client.py

```diff
@@ -24,15 +25,17 @@ def __init__(self, run, input_data, evaluator_name=None, aggregated_metrics=None
         self.aggregated_metrics = aggregated_metrics
 
     def get_result_df(self, exclude_inputs=False):
-        result_df = self.run.result(timeout=BATCH_RUN_TIMEOUT)
+        batch_run_timeout = get_int_env_var(PF_BATCH_TIMEOUT_SEC, PF_BATCH_TIMEOUT_DEFAULT)
+        result_df = self.run.result(timeout=batch_run_timeout)
```

Comment:
why do we need to set the timeout here? As long we set the timeout on the execution part, I think we can omit the setting here?

Reply:
@thy09 to make some comments